src/Resources/ResourceManager/Implementation/Policy/SetAzurePolicyAssignment.cs

```diff
@@ -12,6 +12,8 @@
 // limitations under the License.
 // ----------------------------------------------------------------------------------
 
+using System;
```

Comment:
Move this inside the namespace.

Reply:
The wording is kept consistent to other help strings in this file.